konfera/views.py

```diff
@@ -1,12 +1,28 @@
 from django.shortcuts import redirect
 from django.views.generic import ListView
+from django.shortcuts import render
+from django.contrib import messages
+from django.utils.translation import ugettext_lazy as _
 
 from konfera.models.event import Event
+from konfera.settings import LANDING_PAGE
 
 
 def index(request):
-    latest_conference = Event.objects.published().filter(event_type=Event.CONFERENCE).latest('date_from')
-    return redirect('event_details', slug=latest_conference.slug)
+    try:
+        timewise, event_type = LANDING_PAGE.split('_')
+        timewise = timewise.lower()
+        event_type = event_type.upper()
+    except BaseException as be:
+        raise (_('%s\nIncorrect LANDING_PAGE setting, need latest|earliest_conference|meetup', be))
```

Comment:
I agree, no need for translations of error messages. This will end in log, which will be seen only by admin...